tests/integration/kethaa/test_ethaa.py

```diff
@@ -0,0 +1,103 @@
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../"))
+import pytest
+import web3
+from eth_account._utils.legacy_transactions import (
+    serializable_unsigned_transaction_from_dict,
+)
+from eth_keys import keys
+from starkware.starknet.testing.starknet import Starknet
+from starkware.starkware_utils.error_handling import StarkException
+
+from tests.utils.bits import combine_ints, to_uint
+from tests.utils.signer import BaseSigner, MockEthSigner
+
+txdict = dict(
+    nonce=1,
+    chainId=9001,
```

Comment:
there is a `CHAIN_ID` constant in the project, see https://github.com/sayajin-labs/kakarot/blob/e9114285d06c0dfc017843aa61f90aa48c990592/src/kakarot/constants.cairo#L34